sdk/networkcloud/Azure.ResourceManager.NetworkCloud/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0 (2025-01-31)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
```

Comment:
You will need to keep a section here that fits with what the message says. probably `Other changes`? or `Features added`?

Reply:
got it, fixed!